data/autoyast_qam/12-common_base_installation.xml.ep

```diff
@@ -25,6 +25,9 @@
         % if ($key eq 'ltss') {
         <reg_code><%= $get_var->('SCC_REGCODE_LTSS') %></reg_code>
         % }
+        % if ($key eq 'ltss_es') {
+        <reg_code><%= $get_var->('SCC_REGCODE_LTSS_ES') %></reg_code>
+        % }
```

Comment:
now that you're at it, mind adding LTSS_ES? :D 😅 ( and link it to the LTSS slack thread)